synapse/storage/databases/main/room.py

```diff
@@ -1423,21 +1423,8 @@ def store_room_txn(txn, next_id):
                         "has_auth_chain_index": True,
                     },
                 )
-                if is_public:
-                    self.db_pool.simple_insert_txn(
-                        txn,
-                        table="public_room_list_stream",
-                        values={
-                            "stream_id": next_id,
-                            "room_id": room_id,
-                            "visibility": is_public,
-                        },
-                    )
 
-            async with self._public_room_id_gen.get_next() as next_id:
-                await self.db_pool.runInteraction(
-                    "store_room_txn", store_room_txn, next_id
-                )
+            await self.db_pool.runInteraction("store_room_txn", store_room_txn)
```

Comment:
I think this can be replaced with calling `simple_insert` directly instead of the `txn` version?

Reply:
That was my thought as well - though is there any downside to losing the `store_room_txn` description information?

Reply:
`simple_insert` (and all the `simple_*` functions take a `desc` attribute, which I think is passed as the same parameter to `runInteraction`?

Reply:
I can't believe I missed that - thanks!